### ISSUE TITLE
Can no longer build with latest poppler

### ISSUE THREAD

**Issue by jeroen** [edporras/pdftoedn#5]
The release notes of poppler 0.73 say: 

`         * Remove Gulong, Guint, Gushort, Guchar typedefs`.

When we try to build pdftoedn:

```
clang++ -DHAVE_CONFIG_H -I. -I..      -I/usr/local/Cellar/poppler/0.73.0/include -I/usr/local/Cellar/poppler/0.73.0/include/poppler -I/usr/local/opt/freetype/include/freetype2 -I/usr/local/Cellar/libpng/1.6.36/include/libpng16 -I/usr/local/Cellar/leptonica/1.77.0_1/include/leptonica -I/usr/local/opt/openssl/include -Wno-user-defined-warnings -MT util_versions.o -MD -MP -MF $depbase.Tpo -c -o util_versions.o util_versions.cc &&\
	mv -f $depbase.Tpo $depbase.Po
util_encode.cc:146:27: error: unknown type name 'Guchar'
                          Guchar *pix = img_str->getLine();
                          ^
util_encode.cc:191:27: error: unknown type name 'Guchar'
                          Guchar pix[num_pix_comps];
                          ^
util_encode.cc:314:25: error: unknown type name 'Guchar'
                        Guchar pix;
                        ^
util_encode.cc:318:36: error: use of undeclared identifier 'Guchar'; did you mean 'char'?
                            pix = (Guchar) i;
                                   ^~~~~~
                                   char
util_encode.cc:460:21: error: unknown type name 'Guchar'
                    Guchar pix[num_pix_comps];
                    ^
util_encode.cc:461:21: error: unknown type name 'Guchar'
                    Guchar* mask_bits;
                    ^
util_encode.cc:642:25: error: unknown type name 'Guchar'
                        Guchar* pix = img_str->getLine();
                        ^
7 errors generated.
make[2]: *** [util_encode.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

I think the fix is simply to replace with `unsigned char` like here: https://github.com/freedesktop/poppler/commit/51bb46766cc5b50d81227cd91e518bc78b0a944b

**Comment by edporras** [comment]
Yeah, poppler's been updating their internals on just about every release (I'm subscribed to their developer list). I won't be able to get to this until later today but will push a new release and updated formula as soon as I can. Thanks.

**Comment by jeroen** [comment]
Yes, only the poppler C++/glib/QT api's are stable. Anyway thanks for the quick response.

**Comment by edporras** [comment]
Fixed in v0.36.4: 
https://github.com/edporras/pdftoedn/commit/f4a9a3910f35730286b16514e76da451ca20c567
